packages/store/src/text-adapter.ts

```diff
@@ -181,6 +185,15 @@ export class Text {
       this._yText.meta = { delete: true };
     });
   }
+  // eslint-disable-next-line @typescript-eslint/ban-types
+  cover(index: number, length: number, content: string, attributes?: Object) {
```

Comment:
Maybe `replace` is better to understand?

Reply:
+1 for `replace`